Changes

```diff
@@ -129,6 +129,9 @@ Working version
 
 ### Internal/compiler-libs changes:
 
+- #XXXXX: Added location to exception definitions and type extensions
+  (Samuel Vivien, review by ...)
```

Comment:
You should complete this and the PR is good to go.

Reply:
Should be good now